docs/docs/mdx/getting-started.md

```diff
@@ -40,7 +40,7 @@ can follow these steps for configuring the [gatsby-mdx](/packages/gatsby-mdx/) p
    yarn add gatsby-mdx @mdx-js/mdx @mdx-js/react
```

Comment:
This also isn't part of your updates, but the correct package to use is no longer `gatsby-mdx`, but `gatsby-plugin-mdx`. Can switch that out here. (Other dependencies here are good).